packages/blockchain-extension/tutorials/new-tutorials/basic-tutorials/a3.md

```diff
@@ -92,61 +91,57 @@ Expand the next section of the tutorial to continue.
+In the Deploy Smart Contract form, select "demo-contract@0.0.1" from the drop down list, and click 'Next'.
 
-Private data is an advanced technique for sharing data between organizations. We will not be using that feature for now; it is the subject of a later tutorial.
+<img src="./images/a3.7.png" alt="Select demo-contract"></img>
 
 <img src="./images/bullet.png" alt="[]"></img> &nbsp;&nbsp;&nbsp;&nbsp; `A3.8`: &nbsp;&nbsp;&nbsp;&nbsp;
-In the Command Palette, click 'No' to not provide a private data collection configuration file.
+In step 2 of the form, default values for Definition name and version are provided, click 'Next' to move to Step 3 of the deploy.
 
-<img src="./images/a3.8-a6.17.png" alt="No Private Data Collection configuration file"></img>
+For TypeScript smart contracts, both the name and version are taken from the <i>package.json</i> file in the root of the smart contract project.
 
 The endorsement policy determines which peers get to run the smart contract. As we only have one peer in our organization, we can accept the default.
+Collections configuration and private data is an advanced technique for sharing data between organizations. We will not be using that feature for now; it is the subject of a later tutorial.
 
-<img src="./images/bullet.png" alt="[]"></img> &nbsp;&nbsp;&nbsp;&nbsp; `A3.9`: &nbsp;&nbsp;&nbsp;&nbsp;
-In the Command Palette, click 'Default' to accept the default smart contract endorsement policy.
-
-<img src="./images/a3.9.1-a6.18.1.png" alt="Default endorsement policy"></img>
+<img src="./images/a3.8.png" alt="Click Next on deploy step 2"></img>
 
-Instantiation may take a further few minutes to complete.
-
-When instantiation has completed you will see the new smart contract package listed in the Smart Contracts view, and also underneath the Instantiated section of the connected environment.
-
-<img src="./images/a3.9.2.png" alt="Smart contract view after instantiate"></img>
+<img src="./images/bullet.png" alt="[]"></img> &nbsp;&nbsp;&nbsp;&nbsp; `A3.9`: &nbsp;&nbsp;&nbsp;&nbsp;
+In step 3 of the form, the automated steps of the deploy are sumarized, click 'Deploy' to start the deployment. 
 
-<img src="./images/a3.9.3.png" alt="Fabric environments view after instantiate"></img>
+<img src="./images/a3.9.png" alt="Click Deploy on deploy step 3"></img>
 
-The text after the '@' sign is the version; note that it is set to '0.0.1'. For TypeScript smart contracts, both the name and version are taken from the <i>package.json</i> file in the root of the smart contract project:
+Deployment may take a few minutes to complete.
 
-<img src="./images/a3.9.4.png" alt="Top of package.json"></img>
+When deployment has completed you will see the new smart contract listed under "mychannel" in the Fabric Environments view.
 
-We will update this in a later tutorial.
+<img src="./images/a3.9.1.png" alt="Channel view after deployment"></img>
```

Comment:
`We packaged our smart contract, installed it and instantiated it.` on line 150
should probably be:

`We packaged our smart contract and deployed it.`